src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -1974,6 +1974,31 @@ server.3=zoo3:2888:3888</programlisting>
               </programlisting>
             </listitem>
           </varlistentry>
+
+          <varlistentry>
+            <term>shed</term>
+
+            <listitem>
+              <para>Closes all client connections, forcing them to move to a different server.</para>
+            </listitem>
+          </varlistentry>
+
+          <varlistentry>
+            <term>sh50</term>
+
+            <listitem>
+              <para>Probabilistically closes half of the client connections (i.e. each connection is closed with a probability of 1/2), forcing them to move to a different server.</para>
+            </listitem>
+          </varlistentry>
+
+          <varlistentry>
+            <term>sh50</term>
```

Comment:
Actually this is correct, but the one below (line 1995) is wrong.

It is not trivial to pass the number because 4-letter commands are internally used as 4-byte integer and stored in a static map.
But if we go with ZooKeeperAdmin rather than 4-letter words, looks like we can parametrize.